eng/ci/templates/jobs/run-integration-tests-linux.yml

```diff
@@ -0,0 +1,44 @@
+jobs:
+- job: RunIntegrationTests_Linux
+  displayName: '[Linux] Run Integration Tests'
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-ubuntu-22.04
+    os: linux
+
+  steps:
+  - task: NuGetAuthenticate@1
+    displayName: NuGet Authenticate
+
+  - template: /eng/ci/templates/steps/install-dotnet.yml@self
+
+  - task: DotNetCoreCLI@2
+    displayName: Build DotNetWorker
+    inputs:
+      command: build
+      arguments: '-c Release'
+      projects: DotNetWorker.sln
+
+  - pwsh: |
+      $useIntegrationTestingCoreTools = $null
+      if (-not ([bool]::TryParse($env:USECORETOOLSBUILDFROMINTEGRATIONTESTS, [ref] $useIntegrationTestingCoreTools)))
```

Comment:
Would a template parameter be more fitting for this? You can restrict it to `boolean` with a default value. Can then switch to PowerShellTask and invoke `./setup-e2e-tests.ps1` directly with args